runner/src/mill/runner/MillCliConfig.scala

```diff
@@ -2,309 +2,114 @@ package mill.runner
 
 import mainargs.{Flag, Leftover, arg}
 
-class MillCliConfig private (
+case class MillCliConfig(
     @arg(
       short = 'h',
       doc =
-        """(internal) The home directory of internally used Ammonite script engine;
-           where it looks for config and caches."""
+        """(internal) The home directory where Mill looks for config and caches."""
```

Comment:
I'd rather get rid of it entirely. It's a relict from Ammonite and we only have it still, since it's part exposed in our API (`T.home`). The only thing we put there is the Java `rt.jar` which we generate if missing and share between projects. In essence, we don't look for config here, really.

Reply:
Maybe installing global JVMs could require it?
We have this issue: https://github.com/com-lihaoyi/mill/issues/3480

Reply:
I hid/deprecated it for now